usaspending_api/api_docs/api_documentation/financial_balances/agencies.md

```diff
@@ -0,0 +1,27 @@
+## [Financial Balances](#usaspending-api-documentation)
+**Route:** `/api/v2/financial_balances/agencies`
+
+**Method:** `GET`
+
+This route retrieves financial balance information by funding agency and fiscal year
+
+### Sample Request
+
+`/api/v2/financial_balances/agencies?funding_agency=775&fiscal_year=2017`
+
+### Request Params
+
+* `fiscal_year` - **required** - an integer representing the fiscal year to filter on
+* `funding_agency_id` - **required** - an integer representing the funding agency ID to filter on (should be an agency_id flagged as top tier)
+
+### Response (JSON)
+
+```
+"results": [
```

Comment:
Sample response is missing the `page_metadata` object

Reply:
Done